IdentityCore/src/controllers/broker/ios/MSIDBrokerInteractiveController.m

```diff
@@ -288,7 +288,7 @@ + (BOOL)completeAcquireToken:(nullable NSURL *)resultURL
        brokerResponseHandler:(nonnull MSIDBrokerResponseHandler *)responseHandler
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->